netlify.toml

```diff
@@ -233,6 +233,26 @@
   to = "/docs/cloud/okteto-cli"
   status = 301
 
+[[redirects]]
```

Comment:
Instead of adding new redirects, should we modify the existing ones?
https://github.com/okteto/docs/blob/main/netlify.toml#L56

Reply:
Yes for sure, already working on another commit for that. Wanted to get a preview environment to test since I'm having trouble getting it running locally.

Reply:
@pchico83 Ready for review